plugins/optimization-detective/storage/data.php

```diff
@@ -232,6 +241,7 @@ static function ( $post ): ?array {
 				'version' => wp_get_theme()->get( 'Version' ),
 			),
 		),
+		'active_plugins'   => $active_plugins,
```

Comment:
What is this used for? I don't see any client-side code being changed here that makes use of this data. In other words, why is the list of active plugins now relevant as of this PR?

Reply:
Because the active plugins now is part of the data that goes into the computation of the ETag hash. So when a plugin is activated or deactivated, it will cause the ETag to change.

Reply:
Ah I see. I was misunderstanding the context of this change.